libflux/src/semantic/tests.rs

```diff
@@ -230,6 +334,79 @@ fn instantiation_1() {
     }
 }
 #[test]
+fn imports() {
+    test_infer! {
+        imp: map![
+            "path/to/foo" => package![
+                "a" => "forall [] int",
+                "b" => "forall [] string",
+            ],
+            "path/to/bar" => package![
+                "a" => "forall [] int",
+                "b" => "forall [] {c: int | d: float}",
+            ],
+        ],
+        src: r#"
+            import foo "path/to/foo"
+            import bar "path/to/bar"
+
+            a = foo.a
+            b = foo.b
+            c = bar.a
+            d = bar.b
+        "#,
+        exp: map![
+            "a" => "forall [] int",
+            "b" => "forall [] string",
+            "c" => "forall [] int",
+            "d" => "forall [] {c: int | d: float}",
+        ],
+    }
+    test_infer! {
+        imp: map![
+            "path/to/foo" => package![
+                "f" => "forall [t0] (x: t0) -> t0",
+            ],
+        ],
+        src: r#"
+            import foo "path/to/foo"
+
+            f = foo.f
+        "#,
+        exp: map![
+            "f" => "forall [t0] (x: t0) -> t0",
+        ],
+    }
+    test_infer! {
+        imp: map![
+            "path/to/foo" => package![
+                "f" => "forall [t0] where t0: Addable + Divisible (x: t0) -> t0",
+            ],
+        ],
+        src: r#"
+            import foo "path/to/foo"
+
+            f = foo.f
+        "#,
+        exp: map![
+            "f" => "forall [t0] where t0: Addable + Divisible (x: t0) -> t0",
+        ],
+    }
+    test_infer_err! {
+        imp: map![
+            "path/to/foo" => package![
+                "a" => "forall [] bool",
+                "b" => "forall [] time",
+            ],
+        ],
+        src: r#"
+            import foo "path/to/foo"
+
+            foo.a + foo.b
+        "#,
+    }
```

Comment:
Can you add a simple test case where you do not specify the `as` name of the import? Like `import "path/to/foo"` and then use it as `foo`?